testing/file_test/file_test_base.cpp

```diff
@@ -519,18 +519,27 @@ auto FileTestEventListener::OnTestProgramStart(
   // slowest tests.
   auto print_slowest_tests = absl::GetFlag(FLAGS_print_slowest_tests);
   if (run_count > 1 && print_slowest_tests != 0) {
+    // Sort in a copy so that `FileTestCase` pointers to the original tests
+    // remain stable.
+    llvm::SmallVector<const FileTestInfo*> sorted_tests;
+    sorted_tests.reserve(tests_.size());
+    for (auto& test : tests_) {
+      sorted_tests.push_back(&test);
+    }
```

Comment:
How about
```suggestion
    sorted_tests.append_range(llvm::map_range(tests_, [](const auto& test) { return &test; }));
```

Reply:
Or even:
```suggestion
    sorted_tests.append_range(llvm::make_pointer_range(tests_));
```
You can reduce that further:
```c++
llvm::SmallVector<const FileTestInfo*> sorted_tests = llvm::make_pointer_range(tests_);
```
:)

Reply:
🏌️